features/compose_message.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Compose message
+
+    As a User
+    In order to send a message to someone
+    I want to be able to compose a message
+
+Background:
+    Given following users exists
+        |name   |email              |password   |password_confirmation |
+        |Jenny  |jenny@random.com|password   |password               |
+        |Daniel|daniel@random.com|password|password|
+
+Scenario: Composing a message (Happy Path)
+    Given I am logged in as "Jenny"
+    And I am on the "inbox" page
+    Then I should see "Compose"
```

Comment:
If you are able to click the button it is there, no need for his test

Reply:
Done